bluecode-pay/pom.xml

```diff
@@ -9,7 +9,7 @@
     </parent>
     <groupId>com.example</groupId>
```

Comment:
Would you try to build your project by maven?

Reply:
And don't forget to rewrite liquibase scripts. You have to use syntax of liquibase to create table and not sql syntax.